account/email/verification_service_blackbox_test.go

```diff
@@ -0,0 +1,68 @@
+package email_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/fabric8-services/fabric8-auth/account"
+	"github.com/fabric8-services/fabric8-auth/account/email"
+	"github.com/fabric8-services/fabric8-auth/gormtestsupport"
+	"github.com/fabric8-services/fabric8-auth/test"
+	"github.com/satori/go.uuid"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+type verificationServiceBlackboxTest struct {
+	gormtestsupport.DBTestSuite
+	verificationService email.EmailVerificationService
+	repo                *account.GormVerificationCodeRepository
+}
+
+func TestRunVerificationServiceBlackboxTest(t *testing.T) {
+	suite.Run(t, &verificationServiceBlackboxTest{DBTestSuite: gormtestsupport.NewDBTestSuite()})
+}
+
+func (s *verificationServiceBlackboxTest) SetupTest() {
+	s.DBTestSuite.SetupTest()
+	s.repo = account.NewVerificationCodeRepository(s.DB)
+	s.verificationService = email.NewEmailVerificationClient(s.Application)
+}
+
+func (s *verificationServiceBlackboxTest) TestSendVerificationCodeOK() {
+	identity, err := test.CreateTestIdentity(s.DB, uuid.NewV4().String(), "kc")
+	require.Nil(s.T(), err)
```

Comment:
Let's start using require.Error() and NotError() instead. See https://github.com/fabric8-services/fabric8-wit/pull/1813

Reply:
01322b3 thanks, done in multiple places.